hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -108,7 +109,8 @@ public int getUpsertShuffleParallelism() {
   }
 
   public int getWriteBufferLimitBytes() {
-    return Integer.parseInt(props.getProperty(WRITE_BUFFER_LIMIT_BYTES, DEFAULT_WRITE_BUFFER_LIMIT_BYTES));
+    return Integer
```

Comment:
Can we avoid additional formatting changes across the diff and trim it down to only real changes?